drivers/storage/ebs/executor/ebs_executor.go

```diff
@@ -0,0 +1,216 @@
+package executor
+
+import (
+	"bufio"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/akutz/gofig"
+	"github.com/akutz/goof"
+
+	ebsConfig "github.com/emccode/libstorage/api/drivers/storage/ebs"
```

Comment:
Hi @proudh,

No need to import this twice as separate packages. I'll handle it later. Just calling attention to it for your future projects :)